### PR TITLE
Add more information about file formats for QN

### LINKED ISSUE #50: Add more information about file formats for QN own data
From @dvenprasad:

> For normalizing your own data,  explicitly add what types of files/formats can be plugged into normalize own data script

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Closes #50 and also updates the link to the rendered HTML notebook.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
Added the requirements text from #60. If any changes get suggested over there I can port them over.

File: normalize-own-data/README.md [hunk 1]
```diff
@@ -11,4 +40,10 @@ If you are interested in comparing your own data to data you have downloaded fro
 
 You can read more about the reference distribution and how it is generated in the [quantile normalization section](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization) of our [documentation](http://docs.refine.bio/en/latest/).
 
-In this [example notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/normalize-own-data/quantile_normalize_own_data.Rmd), we demonstrate how to obtain the reference distribution from the refine.bio API and apply it to your own data in R.
+In this [example notebook](https://alexslemonade.github.io/refinebio-examples/normalize-own-data/quantile_normalize_own_data.nb.html), we demonstrate how to obtain the reference distribution from the refine.bio API and apply it to your own data in R.
```
Comment (cansav09) [inline normalize-own-data/README.md]: Normal people might not know what an API is. But also they can probably just fill in the gaps. *shrug*

File: normalize-own-data/README.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+## Requirements and usage
+
+This module requires you to install the following software to run the example yourself:
+
+* [**R**](https://cran.r-project.org/)
+* [**Bioconductor**](https://bioconductor.org/install/)
+* [**tidyverse**](https://www.tidyverse.org/)
+* [**RStudio**](https://www.rstudio.com/products/RStudio/) for working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+* [**devtools**](https://cran.r-project.org/web/packages/devtools/readme/README.html) will be required for installing some packages from GitHub. We recommend installing `devtools` from CRAN.
+* **wget** - [installation instructions for Mac with Homebrew](https://www.maketecheasier.com/install-wget-mac/), [installation instructions for Windows 10](https://builtvisible.com/download-your-website-with-wget/) (under _Install WGET in Windows 10_), [installation instructions for Debian or Ubuntu Linux](https://www.cyberciti.biz/faq/how-to-install-wget-togetrid-of-error-bash-wget-command-not-found/)
```
Comment (cansav09) [inline normalize-own-data/README.md]: Do we `wget` anywhere else that we might need to add these instructions?

Comment (jaclyn-taroni) [reply normalize-own-data/README.md]: Not that I am aware of

File: normalize-own-data/README.md (hunk 1 above)
Comment (jaclyn-taroni) [reply normalize-own-data/README.md]: This is generally intended for a different audience than the differential expression module for example.

Comment (cansav09) [reply normalize-own-data/README.md]: Ah. That's true.

**Pull request merged**